CONTRIBUTING.md

```diff
@@ -1,3 +1,59 @@
+# Contributing
+
+## Getting started
+
 The Contributing page in the wiki provides some guidance on how to get started developing Uyuni.
 
 https://github.com/uyuni-project/uyuni/wiki/Contributing
+
+## Pull Requests
+
+The following roles exist in the context of a Pull Request:
+
+- Creator
+- Reviewer
+- Collaborators
+
+The guidelines noted here describe the expectations that should be met when submitting a Pull request.
+
+### Pull Request Creators
+
+- Please fill out the Pull Request template.
```

Comment:
This feels very repetitive with the "please, please, please". Perhaps we could reword this something along the lines of
```md
Please follow these steps:
 - Fill out the Pull Request template.
 - Provide a description...
```

Reply:
Done. Thanks for the suggestion!